pkg/apis/navigator/validation/elasticsearch.go

```diff
@@ -45,8 +45,7 @@ func ValidateElasticsearchClusterNodePool(np *navigator.ElasticsearchClusterNode
 		el = append(el, field.Invalid(fldPath.Child("replicas"), np.Replicas, "must be greater than zero"))
 	}
 	// TODO: call k8s.io/kubernetes/pkg/apis/core/validation.ValidateResourceRequirements on np.Resources
-	// this will require vendoring kubernetes/kubernetes and switching to use the corev1 ResourceRequirements
-	// struct
+	// this will require vendoring kubernetes/kubernetes.
```

Comment:
❓ Might be simpler with latest version of `dep` which shouldn't need to vendor the entire kubernetes tree....assuming these validation functions don't have too many dependencies.